docroot/sites/settings/cgov_core.settings.php

```diff
@@ -52,6 +52,29 @@
 } else {
   $env = 'local';
 }
+// Settings for metatag attribute cgdp.domain.
+if($is_acsf_env && $acsf_db_name) {
+  // For ACSF env.
+  $acsf_domains = array_keys(gardens_data_get_sites_from_file($GLOBALS['gardens_site_settings']));
+  foreach ($acsf_domains as $domain_value) {
+    if(preg_match('/acsitefactory.com/', $domain_value)){
+      $temp = explode('.', $domain_value);
+      if($temp[1] == $_ENV['AH_SITE_GROUP']){
+        $domain_env = $temp[0];
+      }else {
+        $acsf_env = str_replace('-' . $_ENV['AH_SITE_GROUP'], '', $temp[1]);
+        $domain_env = $temp[0] . '-' . $acsf_env;
+      }
+    }
+  }
+  $settings['cgdp_domain'] = $domain_env;
+}
+elseif (preg_match('/test/', $env)) {
+  $settings['cgdp_domain'] = 'staging';
```

Comment:
If @welshja did not ask for this then we do not have to treat this differently. (i.e. if www-test-ac.cancer.gov says it is test you do not need to replace that with staging.)

Reply:
Removed elseif condition as this is not required anymore, we are printing test in case of test environment.